.babelrc

```diff
@@ -0,0 +1,4 @@
+{
```

Comment:
Problaly it would be best to leave this. This file is needed when running 'gulp js' to specify dependencies required to run babelify so everytime setuping the project this file would be needed.